index.md

```diff
@@ -35,6 +35,125 @@ Ruby Unconf Hamburg 2019 will happen on 25th and 26th of May 2019 at the [HAW Ha
   </div>
 </div>
 
+<div class="content-section content-section--whitebg" markdown="1">
+## Agenda
+
+**[Real-time schedule](https://docs.google.com/spreadsheets/d/1YLiU9QU6HH_MaRT-zh_yJf6pQfUdYk59plJUwaOWciE/edit?usp=sharing)**
```

Comment:
It would be super nice if you can style the link like a button. You can have a look at the menu button as inspiration. It would be totally fine if you want take care of this in a follow up pull request.

Reply:
will do later